README.md

```diff
@@ -22,9 +22,7 @@ We don't use the Bootstrap 5 ruby gem here, as it runs on a
 
 ### Running Locally
 
-To run the site locally, you'll first need to update the [Gemfile](Gemfile) by uncommenting the `jekyll` gem and commenting out the `github-pages` gem.
-
-You'll then need to download the Bootstrap 5 Sass sources by running [local_setup.sh](local_setup.sh).
+To run the site locally, you'll first need to download the Bootstrap 5 Sass sources by running [local_setup.sh](local_setup.sh).
```

Comment:
This file is empty in the PR at the moment.

Reply:
[The file isn't empty on the PR branch](https://github.com/gracegrimwood/kroxylicious.github.io/blob/update-jekyll-deps/local_setup.sh), but it looks empty in the PR diff because the only difference is file mode bits (`0644`/`-rw-r--r--` in the main branch, `0755`/`-rwxr-xr-x` in the PR branch). Git sees this as being an "empty file" even though it isn't, because it has trouble representing a diff with only file bit changes and no content difference.